pkg/broker/dynamicplans/plan.go

```diff
@@ -74,3 +76,24 @@ func (p Plan) String() string {
 
 	return string(s)
 }
+
+func (p *Plan) UnmarshalJSON(data []byte) error {
+	type plan Plan
+
+	var pl plan
+	err := json.Unmarshal(data, &pl)
+
+	// try to fix the plan in case of an error
+	if err != nil && pl.APIKey != nil {
+		if pl.Project.OrgID == "" {
+			return fmt.Errorf("failed to fix the plan with error: %w", err)
+		}
+
+		pl.APIKey["orgID"] = pl.Project.OrgID
+		pl.APIKey["orgId"] = pl.Project.OrgID
```

Comment:
`orgId` is always wrong here - none of the data in Realm has it

Reply:
Yeah, my mistake. I thought I saw it being used.

Reply:
`orgId` is not automatically converted for maps, so I thought I need to assign both.
Now I just always convert to `orgID` during Unmarshaling.

Reply:
Cause it is a map, maybe it is better to lowercase everything at the beginning to avoid similar issues.
Potential change for the future.

Reply:
Not sure if you mean a schema change or a code change there -- but either way, the code looks ok now